tests/resources/aria-at-harness.mjs

```diff
@@ -162,10 +176,107 @@ function executeScriptInTestPage() {
   }
 }
 
+/**
+ * @typedef AT
+ * @property {string} name
+ * @property {string} key
+ */
+
+/**
+ * @typedef Support
+ * @property {AT[]} ats
+ * @property {{system: string[]}} applies_to
+ * @property {{directory: string, name: string}[]} examples
+ */
+
+/**
+ * @typedef {{[mode in ATMode]: {[assitiveTech: string]: string;};}} CommandsAPI_ModeInstructions
+ */
+
+/**
+ * @typedef {([string] | [string, string])[]} CommandAT
+ */
+
+/**
+ * @typedef {{[atName: string]: CommandAT}} CommandMode
+ */
+
+/**
+ * @typedef Command
+ * @property {CommandMode} [reading]
+ * @property {CommandMode} [interaction]
+ */
+
+/**
+ * @typedef {{[commandDescription: string]: Command}} Commands
+ */
+
+/**
+ * @callback CommandsAPI_getATCommands
+ * @param {ATMode} mode
+ * @param {string} task
+ * @param {string} assistiveTech
+ * @returns {string[]}
+ */
+
+/** @typedef {"reading" | "interaction"} ATMode */
+
+/**
+ * @callback CommandsAPI_getModeInstructions
+ * @param {ATMode} mode
+ * @param {string} assistiveTech
+ * @returns {string}
+ */
+
+/**
+ * @callback CommandsAPI_isKnownAT
+ * @param {string} assistiveTech
+ * @returns {AT}
+ */
+
+/**
+ * @typedef CommandsAPI
+ * @property {Commands} AT_COMMAND_MAP
+ * @property {CommandsAPI_ModeInstructions} MODE_INSTRUCTIONS
+ * @property {Support} support
+ * @property {CommandsAPI_getATCommands} getATCommands
+ * @property {CommandsAPI_getModeInstructions} getModeInstructions
+ * @property {CommandsAPI_isKnownAT} isKnownAT
+ */
+
+/**
+ * @typedef BehaviorJSON
+ * @property {string} setup_script_description
+ * @property {string} setupTestPage
```

Comment:
This is the only property which isn't snake_case! Maybe there's a good reason for that but I can't tell!

Reply:
This type is the format of the test json files the test generation creates. So I think changing that key is off limits without a larger conversation.